src/core/Currency.cpp

```diff
@@ -0,0 +1,31 @@
+#include "Currency.h"
+
+Currency::Currency(double cents) : m_Cents(qRound(cents * 100.0)) {}
+Currency::Currency(const Currency& other) : m_Cents(other.m_Cents) {}
```

Comment:
I'm not in love with one-liners for the sake of one-liners. Multiple lines don't hurt!

Reply:
Fixed